models/graphql/shopify_gql__customers.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(enabled=var('shopify_api', 'rest') == var('shopify_api_override','graphql')) }}
+
+with customers as (
+
+    select 
+        {{ dbt_utils.star(from=ref('int_shopify_gql__customer'), except=["orders_count", "total_spent"]) }}
+    from {{ ref('int_shopify_gql__customer') }}
+
+), orders as (
+
+    select *
+    from {{ ref('shopify_gql__customers__order_aggregates' )}}
+
+{% if var('shopify_using_abandoned_checkout', True) %}
```

Comment:
Same comment about this likely needing to be `shopify_gql_using_abandoned_checkout`